src/styles/blocks/header.scss

```diff
@@ -0,0 +1,60 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  height: $nav-hight;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px 0 #0000000d;
+
+  &__link {
+    display: flex;
+  }
+
+  &__nav {
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+
+    &-list {
+      display: flex;
+      flex-wrap: wrap;
+      gap: 20px;
+      list-style: none;
+      line-height: 14.6px;
+      color: $nav-link-color;
+      padding: 0;
+    }
+
+    &-link {
+      display: flex;
+      position: relative;
+      align-items: center;
+      height: $nav-hight;
```

Comment:
The same typo is present here as well. Please correct `$nav-hight` to `$nav-height` to ensure consistency and correct spelling.